### DISCUSSION TITLE
How to transfer or duplicate repo's fork issues to repo issues

### DISCUSSION THREAD

**Discussion by volkantash** [community/community#4480] [Projects and Issues]
How to transfer or duplicate repo's fork issues to repo issues? If I could push the fork issues to the main-repo without transferring them, that would be a much more convenient method. Because the issues maintained from the fork will be accessible from the main-repo. And if we can see the main-repo issues in the fork, we will be able to create a working network without going elsewhere while working on the fork.

Sorry for confusing the issue. I hope you can understand. :)

**Comment by inson1** [comment]
Yes we need that

**Comment by dragon-lex** [comment]
I'm waiting for this feature too

**Comment by jordantrizz** [comment]
Also waiting on this feature!

**Comment by rdmark** [comment]
Yes please! I have a bunch of fork issues that I need migrated to the mother repo.
